tests/unit/debounce-task-test.js

```diff
@@ -15,10 +15,6 @@ module('ember-lifeline/debounce-task', function (hooks) {
     };
   });
 
-  hooks.afterEach(function () {
-    runDisposables(this.obj);
```

Comment:
We still need to do this conceptually, but its probably `import { destroy } from '@ember/destroyable'; destroy(this.obj);`

Reply:
Hmmm. OK. I can update them.